### DISCUSSION TITLE
calibrate the lambda

### DISCUSSION THREAD

**Discussion by TIANRENK** [nkaz001/hftbacktest#33] [Q&A]
I followed the example in the document  caculated the lambda of  Guéant–Lehalle–Fernandez-Tapia Market Making Model. I found the market order’s arrivals  does not match the Poisson distribution.  How can I  better calibrate the lambda .

**Comment by TIANRENK** [comment]
> I followed the example in the document caculated the lambda of Guéant–Lehalle–Fernandez-Tapia Market Making Model. I found the market order’s arrivals match the Poisson distribution. How can I better calibrate the lambda .

The  market order's hits  just 0.5 ticksize over 85%.

**Comment by nkaz001** [comment]
I've also noticed the same phenomenon, but I lack a definite answer. As you pointed out, a non-parametric approach seems necessary. One straightforward method that occurs to me involves using the median or decile to directly measure order arrival, leading to an empirical derivation of half spread and skew. For instance, half_spread could be calculated as a * decile(order_arrival, d) + b, with skew described similarly. Starting with an identity function(a = 1 and b = 0) would offer the simplest initial approach.
